index.html

```diff
@@ -265,6 +266,51 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
   }
 }
         </pre>
+NOTE TO EDITORS. Please leave the example from the Latest Published Version here as is.
+    <p>
+    Example 1 depicts a simple alumni verifiable credential. 
+    </p>
+NOTE TO EDITORS. Please insert the 5 tabbed example from the editors draft here.
+    <p>
+    The tab labelled "Committed" indicates that the properties <code>id</code>, <code>type</code>, 
+    <code>credentialSchema.id</code>, <code>credentialSchema.type</code>,
+    <code>credentialSubject.id</code> and <code>credentialSubject.degree.type</code> 
+    will be selectively disclosable. The properties <code>@context</code>, <code>issuer</code>, 
+    <code>validFrom</code> and <code>credentialSubject.degree.name</code> will always be disclosed.
```

Comment:
```suggestion
    <code>credentialSubject.id</code>, and <code>credentialSubject.degree.type</code> 
    will be selectively disclosable. The properties <code>@context</code>, <code>issuer</code>, 
    <code>validFrom</code>, and <code>credentialSubject.degree.name</code> will always be disclosed.
```

Reply:
I guess I am just old fashioned as I was taught not to add the Oxford comma :-)

Reply:
AFAIK the W3C style requires the Oxford comma :-) And you are right about consistency.

(This seems to be a religious issue for native English speakers... If I remember well, although it was a _very_ long time ago, when I learned English I was advised to use it.)